Scripts/Python/xInvite.py

```diff
@@ -172,7 +172,7 @@ def DeleteInvitation(params=None):
     return PtGetLocalizedString("KI.Invitation.DeletedInvitation") + str(passkey)
 
 def MeChat(params=None):
-    if (params == None):
+    if (params == None or params == "" or params.isspace()):
```

Comment:
```suggestion
    if (not params or params.isspace()):
```

Empty strings are Falsey in Python.

Reply:
Updated according to your suggestions